x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.test.tsx

```diff
@@ -680,65 +680,6 @@ describe('rule_form', () => {
       expect(wrapper.find('[data-test-subj="ruleFormConsumerSelect"]').exists()).toBeFalsy();
     });
 
-    it('Do not show alert query in action when multi consumer rule type does not have a consumer selected', async () => {
```

Comment:
@XavierM this PR makes this test fail, but I'm thinking about removing it instead of updating. We updated the behavior of the consumer selection UI so that it automatically selects the first available consumer, so this isn't really a state that the UI can get into. What do you think?

Reply:
I think it is safe to remove since most of our rule type are using AAD